schedule/yam/agama_auto_patterns.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
please replace `auto` by `unattended` in this file and anything new you find that we are using.

Reply:
updated.